CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ New deprecation(s):
 
 ### Other
 
+- **General**: Fixed a typo in the StatefulSet scaling resolver (N/A)
```

Comment:
I think that entry log isn't required as it's just a quick correction on a message. If you still want to maintain the changelog (which is also correct), please add the pull request url instead of (N/A)

Reply:
cool, added a link to the PR.